org.upscayl.Upscayl.yml

```diff
@@ -0,0 +1,64 @@
+id: org.upscayl.Upscayl
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: upscayl-run
+separate-locales: false
+
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+
+modules:
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+  - name: upscayl
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: >-
+          https://github.com/upscayl/upscayl/releases/download/v2.5.0/upscayl-2.5.0-linux.zip
+        sha256: 6e8a761c76f06daf58128ac763c1d8b81811aa3b8ba1f75026e861b32f09363d
+      - type: file
+        path: org.upscayl.Upscayl.metainfo.xml
+      - type: script
+        dest-filename: upscayl-run
+        commands:
+          - zypak-wrapper /app/upscayl/upscayl
+    build-commands:
+      - install -d /app/upscayl
+      - unzip upscayl-2.5.0-linux -d /app/upscayl
+      - install upscayl-run /app/bin/
+      - >-
+        install -Dm644 /app/upscayl/icon_512x512.png
+        /app/share/icons/hicolor/512x512/apps/org.upscayl.Upscayl.png
+      - >-
+        install -Dm644 /app/upscayl/icon_128x128.png
+        /app/share/icons/hicolor/128x128/apps/org.upscayl.Upscayl.png
+      - >-
+        install -Dp -m644 org.upscayl.Upscayl.metainfo.xml
+        /app/share/metainfo/org.upscayl.Upscayl.metainfo.xml
+      - >-
+        touch /app/upscayl/org.upscayl.Upscayl.desktop
```

Comment:
I don't see why this is done, and in that path.

Reply:
we do not have a desktop file, so we make an empty file and configure that before installing it

Reply:
then put the .desktop here until you ship it with the tarball. A desktop app without .desktop it's a bug. it's a 20 yr old standard.

Reply:
alright will do

Reply:
> then put the .desktop here until you ship it with the tarball. A desktop app without .desktop it's a bug. it's a 20 yr old standard.

updated, we ship the desktop file inside the zip now